homeassistant/components/toon/const.py

```diff
@@ -192,7 +196,7 @@
         ATTR_SECTION: "power_usage",
         ATTR_MEASUREMENT: "day_average",
         ATTR_UNIT_OF_MEASUREMENT: ENERGY_KILO_WATT_HOUR,
-        ATTR_DEVICE_CLASS: None,
+        ATTR_DEVICE_CLASS: DEVICE_CLASS_ENERGY,
         ATTR_ICON: "mdi:power-plug",
```

Comment:
We shouldn't set custom icons when setting device class.

Reply:
There you go: https://github.com/home-assistant/core/pull/38768